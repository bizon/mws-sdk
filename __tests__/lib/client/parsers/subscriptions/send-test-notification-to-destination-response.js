const parseXml = require('../../../../../lib/client/parsers')
const parseSendTestNotificationToDestinationResponse = require('../../../../../lib/client/parsers/subscriptions/send-test-notification-to-destination-response')

describe('lib.client.parsers.subscriptions.send-test-notification-to-destination-response', () => {
  it('should parse the SendTestNotificationToDestination example response from MWS doc', () => {
    const doc = parseXml(
      `<?xml version="1.0"?>
      <SendTestNotificationToDestinationResponse xmlns="http://mws.amazonservices.com/schema/Subscriptions/2013-07-01">
        <SendTestNotificationToDestinationResult />
        <ResponseMetadata>
          <RequestId>f662dae6-bde0-4e75-a53b-741abEXAMPLE</RequestId>
        </ResponseMetadata>
      </SendTestNotificationToDestinationResponse>`,
    )

    const res = parseSendTestNotificationToDestinationResponse(
      '/subscriptions:SendTestNotificationToDestinationResponse',
      doc,
    )

    expect(res).toMatchSnapshot()
  })
})
