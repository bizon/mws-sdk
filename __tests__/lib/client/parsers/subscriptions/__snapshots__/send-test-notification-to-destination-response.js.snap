// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lib.client.parsers.subscriptions.send-test-notification-to-destination-response should parse the SendTestNotificationToDestination example response from MWS doc 1`] = `
{
  "responseMetadata": {
    "requestId": "f662dae6-bde0-4e75-a53b-741abEXAMPLE",
  },
  "sendTestNotificationToDestinationResult": {},
}
`;
